mpv.el

```diff
@@ -362,6 +383,18 @@ of \\[universal-argument] will add another `mpv-seek-step' seconds."
   (interactive)
   (mpv--enqueue '("revert-seek") #'ignore))
 
+;;;###autoload
+(defun mpv-playlist-next ()
+  "Go to the next entry on the playlist."
+  (interactive)
+  (mpv--enqueue '("playlist-next" "weak") #'ignore))
+
+;;;###autoload
+(defun mpv-playlist-prev ()
+  "Go to the previous entry on the playlist."
+  (interactive)
+  (mpv--enqueue '("playlist-prev" "weak") #'ignore))
+
```

Comment:
I think "weak" is the default? If this is different for your version of mpv, we can also keep it.
```suggestion
;;;###autoload
(defun mpv-playlist-next ()
  "Go to the next entry on the playlist."
  (interactive)
  (mpv--enqueue '("playlist-next") #'ignore))

;;;###autoload
(defun mpv-playlist-prev ()
  "Go to the previous entry on the playlist."
  (interactive)
  (mpv--enqueue '("playlist-prev") #'ignore))

```

Reply:
@kljohann 
On Ubuntu20.04LTS, the "weak" is default value.

       playlist-next <flags>
              Go to the next entry on the playlist.

              First argument:

              weak (default)
                     If the last file on the playlist is currently played, do nothing.

              force  Terminate playback if there are no more files on the playlist.

       playlist-prev <flags>
              Go to the previous entry on the playlist.

              First argument:

              weak (default)
                     If the first file on the playlist is currently played, do nothing.

              force  Terminate playback if the first file is being played.